features/content_profile.feature

```diff
@@ -1265,7 +1265,7 @@ Feature: Content Profile
                     },
                     "type": "list",
                     "required": true,
-                    "mandatory_in_list": {"scheme": {"category": "category"}}
+                    "mandatory_in_list": {"scheme": {"category": {"required": true}}}
```

Comment:
I guess such change would require upgrade script, wouldn't it?

Reply:
Oops, surely there should be one. Will write one.